README.md

```diff
@@ -59,12 +59,18 @@ See the [config for more options](https://github.com/riemers/ansible-gitlab-runn
 Example Playbook
 ----------------
 ```yaml
-- hosts: all
-  remote_user: root
+- hosts: gitlab-runner
+  become: yes
   vars_files:
     - vars/main.yml
   roles:
     - { role: riemers.gitlab-runner }
+  vars:
+    proxy_env:
+      http_proxy: 'http://proxy-squid:3128'
```

Comment:
may be use?

[gitlab-runner:vars]
http_proxy=http://proxy-squid:3128
https_proxy=http://proxy-squid:3128

https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html

i check and write later

Reply:
We should at least place it in such a way that it is optional. Since the majority of users is not using a proxy. Like comment it out in the example, with a note.

But if you can show me that it works for you, i can merge it since i don't use a proxy.